tests/new_unit_tests/Commands/Multidev/MergeFromDevCommandTest.php

```diff
@@ -0,0 +1,96 @@
+<?php
+
+namespace Pantheon\Terminus\UnitTests\Commands\Multidev;
+
+use Pantheon\Terminus\Commands\Multidev\MergeFromDevCommand;
+
+/**
+ * Testing class for Pantheon\Terminus\Commands\Multidev\MergeFromDevCommand
+ */
+class MergeFromDevCommandTest extends MultidevCommandTest
+{
+    /**
+     * @inheritdoc
+     */
+    protected function setUp()
+    {
+        parent::setUp();
+
+        $this->command = new MergeFromDevCommand($this->getConfig());
+        $this->command->setLogger($this->logger);
+        $this->command->setSites($this->sites);
+        $this->environment->method('mergeFromDev')->willReturn($this->workflow);
+    }
+
+    /**
+     * Tests the multidev:merge-from-dev command
+     */
+    public function testMultidevDelete()
+    {
+        $this->environment->id = 'multipass';
+
+        $this->logger->expects($this->once())
+            ->method('log')
+            ->with(
+                $this->equalTo('notice'),
+                $this->equalTo("Merged the dev environment into {env}."),
+                $this->equalTo(['env' => $this->environment->id,])
+            );
+        $this->workflow->expects($this->once())
+          ->method('wait');
+        $this->environment->expects($this->once())
+          ->method('mergeFromDev')
+          ->with($this->equalTo(['updatedb' => false,]));
+        $this->workflow->method('isSuccessful')->willReturn(true);
+
+        $out = $this->command->mergeFromDev("site.{$this->environment->id}");
+        $this->assertNull($out);
+    }
+
+    /**
+     * Tests to ensure the multidev:merge-from-dev to ensure it passes the 'updatedb' option successfully
+     */
+    public function testMultidevDeleteWithBranch()
+    {
+        $this->environment->id = 'multipass';
+
+        $this->logger->expects($this->once())
+            ->method('log')
+            ->with(
+                $this->equalTo('notice'),
+                $this->equalTo("Merged the dev environment into {env}."),
+                $this->equalTo(['env' => $this->environment->id,])
+            );
+        $this->workflow->expects($this->once())
+            ->method('wait');
+        $this->environment->expects($this->once())
+            ->method('mergeFromDev')
+            ->with($this->equalTo(['updatedb' => true,]));
+        $this->workflow->method('isSuccessful')->willReturn(true);
```

Comment:
I think I must be guilty of this myself now that I think of it but this test might be easier to follow if expectations were in procedural order?
```
$this->environment->expects()->method('mergeFromDev')
$this->workflow->expects()->method('wait')
$this->workflow->method('isSuccessful')
$this->logger->expects()->method('log')
```